packages/cli/src/credentials/credentials.controller.ee.ts

```diff
@@ -134,50 +134,56 @@ EECredentialsController.post(
  * Grant or remove users' access to a credential.
  */
 
-EECredentialsController.put('/:credentialId/share', async (req: CredentialRequest.Share, res) => {
-	const { credentialId } = req.params;
-	const { shareWithIds } = req.body;
+EECredentialsController.put(
+	'/:credentialId/share',
+	ResponseHelper.send(async (req: CredentialRequest.Share, res) => {
+		const { credentialId } = req.params;
+		const { shareWithIds } = req.body;
+
+		if (
+			!Array.isArray(shareWithIds) ||
+			!shareWithIds.every((userId) => typeof userId === 'string')
+		) {
+			return res.status(400).send({ message: 'Bad request' });
```

Comment:
This still needs to be
```suggestion
			return res.status(400).json({ message: 'Bad request' });
```

Reply:
nevermind. [looks like like `.send` now internally calls `.json` for objects.](https://github.com/expressjs/express/blob/master/lib/response.js#L162)